app/api/tax.py

```diff
@@ -43,9 +66,41 @@ class Meta:
                          type_='event')
 
 
-class TaxList(ResourceList):
+class TaxListPost(ResourceList):
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    def before_get(self, args, kwargs):
+        if 'Authorization' in request.headers:
+            if has_access('is_admin'):
```

Comment:
I think this can be changed to be written to write like this:

```python
if 'Authorization' in request.headers and has_access('is_admin'):
    self.schema = TaxSchema
else:
    self.schema = TaxSchemaPublic
```